server/app/Module.scala

```diff
@@ -0,0 +1,9 @@
+import com.google.inject.AbstractModule
+import com.google.inject.name.Names.named
+
+import scala.util.Random.nextInt
+
+class Module extends AbstractModule {
+  override def configure() =
```

Comment:
I do not like the idea with ImplemenetedBy annotation. Now, in order to see which components I create with dependency injection framework, I need to run over all classes in project and see what it is available. Having this binding inside the Module class is much better. There is no significant win to use annotations. Why did you move this to annotations?